app/assets/stylesheets/header.css.scss

```diff
@@ -0,0 +1,34 @@
+@import "bootstrap";
+
+.navbar-brand { position: relative; z-index: 2; }
```

Comment:
Honestly I have no clue, but I've done it in the past and no one has complained haha.